class/SyncHandlers/SyncHandlerBase.php

```diff
@@ -61,6 +66,11 @@ public static function delete_many(array $shortcodes)
     }
   }
 
+  public static function delete_one(string $shortcode)
+  {
+    static::delete_many(array($shortcode));
```

Comment:
We should use the cleaner `[]` syntax for arrays instead of `array()`

```suggestion
    static::delete_many([$shortcode]);
```